README.md

```diff
@@ -38,6 +38,16 @@ Example Geology Query
 Example Physiography Query
  - http://localhost:5000/physiography/64.606/-147.345
 
+## Updating the JSON data on the API
+
+We have an endpoint specifically meant to pull down data
```

Comment:
This is good; I suspect that there's been a bit of a change in other concurrent PRs to make this happen automatically (unsure); also it'd be good to indicate here that this endpoint is called against the production API via an Action when code is merged into the geospatial vector veracity repo.

Reply:
x-ref #145, #146